README.md

```diff
@@ -19,11 +19,20 @@ using Dates
 using TimeZones
 import TimeZones: ZonedDateTime
 market = ElectricityMarketData.PjmMarket()
-df_raw = ElectricityMarketData.get_day_ahead_lmp(
-    market,
-    ZonedDateTime(DateTime(2024, 1, 1, 0, 0), tz"UTC-4"),
-    ZonedDateTime(DateTime(2024, 1, 1, 1, 0), tz"UTC-4"),
-)
+ElectricityMarketData.get_pjm_lmp_raw_data(market,
```

Comment:
Maybe keeps the default api in the readme

Reply:
I will change it. I forgot to change it

Reply:
up